store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+                    array_values.append({"booleanValue": item})
+                else:
+                    array_values.append({"stringValue": str(item)})
+            firestore_fields[key] = {"arrayValue": {"values": array_values}}
+        else:
+            firestore_fields[key] = {"stringValue": str(value)}
+    
+    return firestore_fields
+
+
+def create_cloudevent_json(project_id: str, database_id: str, collection_path: str, 
+                          document_id: str, document_fields: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)